sfaira/data/base.py

```diff
@@ -903,6 +900,29 @@ def write_meta(
 
     # Properties:
 
+    @property
+    def id(self) -> str:
+        def clean(s):
+            return s.replace(' ', '').replace('-', '').replace('_', '').lower()
+
+        if hasattr(self, 'idx'):
+            idx = self.idx
+        else:
+            idx = "000"
```

Comment:
Create starts with 001 and not 000. The already existing dataloaders that I saw also started with 001 I think

https://github.com/theislab/sfaira/blob/dev/sfaira/commands/create_dataloader.py#L122